providers/implementations/include/prov/implementations.h

```diff
@@ -536,6 +536,33 @@ extern const OSSL_DISPATCH ossl_x448_to_SubjectPublicKeyInfo_pem_encoder_functio
 extern const OSSL_DISPATCH ossl_x448_to_OSSL_current_der_encoder_functions[];
 extern const OSSL_DISPATCH ossl_x448_to_text_encoder_functions[];
 
+extern const OSSL_DISPATCH ossl_ml_dsa_44_to_EncryptedPrivateKeyInfo_der_encoder_functions[];
```

Comment:
FWIW, the `ML-KEM` declarations are in `#ifdef OPENSSL_NO_ML_KEM` though I guess it is harmless to declare prototypes that are unused.

Reply:
Must be since the file is full of them :)